multimedia/ffmpeg4/Makefile

```
@@ -0,0 +1,343 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ffmpeg4
+PKG_VERSION:=4.1.4
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Maintained by OpenWrt community
```

I would rather to see it empty here.

```suggestion
PKG_MAINTAINER:=
```

---

Done, thanks